### PR TITLE
Create basic readme file

### LINKED ISSUE #11: Proposal of a README.md Template Model
I've been searching for good and clean readme.md templates for open source projects. Bellow is the one I found more interesting. What you guys think?

https://gist.github.com/PurpleBooth/109311bb0361f32d87a2

### PR DESCRIPTION
**Pull request by wellyal**
resolve #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,74 @@
...
+```
+
+### Prerequisites
+
+You should have at least the LTS node version 6.11. We recommend you to use [nvm](https://github.com/creationix/nvm) it will make easy for you to switch between node/mpm versions.
+
+
+### Installing
+
+Run `npm start` (inside you clone folder) to run our application over a local server.
```
Comment (italopessoa) [inline README.md]: Run `npm start` (into the project folder) to run the application over a local server

the application isn't ours 😃

File: README.md [hunk 2]
```diff
@@ -1,3 +1,74 @@
...
+
+* [React](https://facebook.github.io/react/) - A JavaScript library for building user interfaces
+* [Redux](http://redux.js.org/) - The web framework used
+* [Apollo Client](http://www.dropwizard.io/1.0.2/docs/) - Redux is a predictable state container for JavaScript apps
+* [Styled Components](http://redux.js.org/) - The web framework used
+* [Jest](http://www.dropwizard.io/1.0.2/docs/) - A library to unit test javascript code
+
+## Contributing
+
+Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
```
Comment (italopessoa) [inline README.md]: **about** our code...

File: README.md [hunk 3]
```diff
@@ -1,3 +1,74 @@
-<h1 align="center">
-    Inside
-</h1>
+# Inside
+
+Inside is an open source plataform to deal with all the intern process of a company related to its employees. Our goal is to make easy to keep a track and manage all information related to the people that works **inside** the company.
```
Comment (italopessoa) [inline README.md]: platform
people that **work**

**Review by italopessoa — CHANGES_REQUESTED** [review]
Very good @wellyal 
Just a typo and some suggestions.

File: README.md (hunk 2 above)
Comment (wellyal) [reply README.md]: @italopessoa in this case it was the same default text of the template we are using. I didn't changed the original text. https://gist.github.com/PurpleBooth/109311bb0361f32d87a2

Comment (italopessoa) [reply README.md]: So looks like I've learned something here 🐴

**Review by italopessoa — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,3 +1,74 @@
...
+### Installing
+
+Run `npm start` (into the project folder) to run the application over a local server.
+
+```
+cd inside-client
+npm start
+```
+
+## Built With
```
Comment (jouderianjr) [inline README.md:39]: I think this section is not correct, we have some incorrect links (Apollo client, Styled Components, Jest) and some incorrect descriptions

**Review by jouderianjr — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (wellyal) [reply README.md:39]: Thanks for figuring this out. My clipboard must have gone crazy. I fixed, can you check if everything is ok now?

**Review by jouderianjr — APPROVED** [review]
Awesome 💃

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1 @@
+## Contributing
```
Comment (jvcjunior) [inline CONTRIBUTING.md:1]: The issue is about creating a basic readme file, I don't know if this file should be here. What do you think @jouderianjr @adrianobrito ? Maybe could be in another commit.

File: README.md [hunk 6]
```diff
@@ -1,3 +1,74 @@
...
+```
+
+### Prerequisites
+
+You should have at least the LTS node version 6.11. We recommend you to use [nvm](https://github.com/creationix/nvm) it will make easy for you to switch between node/mpm versions.
+
+
+### Installing
+
+Run `npm start` (into the project folder) to run the application over a local server.
```
Comment (jvcjunior) [inline README.md:32]: Does this code calls `npm install`?

File: CONTRIBUTING.md (hunk 5 above)
Comment (adrianobrito) [reply CONTRIBUTING.md:1]: I think that's a good idea keep this file in repository. It make easy to understand the necessary guidelines to external collaborators. It's fairly used pratice in majors projects like [Scala language](https://github.com/scala/scala/blob/2.12.x/CONTRIBUTING.md) and [Atom editor](https://github.com/atom/atom/blob/master/CONTRIBUTING.md).

However, It's necessary to discuss with the team members to ellaborate the content of this file.

**Review by adrianobrito — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (italopessoa) [reply README.md:32]: I don't think so @jvcjunior
